<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SectoresGUI</class>
 <widget class="QDialog" name="SectoresGUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>396</width>
    <height>363</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Sectores o Dependencias</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QTableView" name="tablaSectores"/>
   </item>
   <item row="0" column="1">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QToolButton" name="btmIngresar">
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="../ContablesPrincipal.qrc">
         <normaloff>:/iconos/mas.png</normaloff>:/iconos/mas.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="btmEliminar">
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="../ContablesPrincipal.qrc">
         <normaloff>:/iconos/menos.png</normaloff>:/iconos/menos.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="btmModificar">
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="../ContablesPrincipal.qrc">
         <normaloff>:/iconos/default-applications-capplet.png</normaloff>:/iconos/default-applications-capplet.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btmSeleccionar">
       <property name="text">
        <string>Seleccionar</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btmCancelar">
       <property name="text">
        <string>Cancelar</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../ContablesPrincipal.qrc"/>
 </resources>
 <connections/>
</ui>
